scss/_popover.scss

```diff
@@ -100,6 +100,9 @@
       }
     }
   }
+  &.inModal {
```

Comment:
Our CSS class naming convention is `dash-separated-words`, not `camelCase`.

I advise renaming this to `.popover-in-modal` and un-nesting it.